### PR TITLE
chore: better report auto-generated diff

### PR DESCRIPTION
**Pull request by johnbieren**

### CODE REVIEW

**Review by scoheb — APPROVED** [review]

File: .github/workflows/kube-linter.yaml [hunk 1]
```diff
@@ -30,6 +30,7 @@ jobs:
             if [[ $(git status --porcelain) ]]; then
                 echo "Did you forget to build the manifests locally?" >&2
                 echo "Please run build-manifests.sh and update your PR" >&2
+                git diff >&2
```
Comment (scoheb) [inline .github/workflows/kube-linter.yaml:33]: actually does this cause new files not added to show up in output?

Comment (johnbieren) [reply .github/workflows/kube-linter.yaml:33]: I'm not sure I am understanding what you mean

Comment (scoheb) [reply .github/workflows/kube-linter.yaml:33]: does this cover the cause where a file was not checked into git ... as is the common cause of this issue.

Comment (johnbieren) [reply .github/workflows/kube-linter.yaml:33]: So like an entire file is missing from the `auto-generated` dir?

Comment (scoheb) [reply .github/workflows/kube-linter.yaml:33]: yes. there be some `git diff` options needed. but we can improve overtime.

Comment (johnbieren) [reply .github/workflows/kube-linter.yaml:33]: You're right, I tested it on my other PR and if a file is missing, this extra line doesn't print anything out, so you are left with not knowing like before

**Pull request merged**